packages/graph/hash_graph/lib/graph/src/store/postgres/context/mod.rs

```diff
@@ -73,7 +73,7 @@ impl<C: AsClient> PostgresContext for PostgresStore<C> {
     {
         ontology::read_versioned_type(&self.client, uri)
             .await
-            .attach_printable("could not read ontology type")
+            .attach_printable_lazy(|| format!("could not read ontology type: {uri}"))
```

Comment:
I'd prefer if we added the URI in a different attachment as it means we could use it later on.